src/gossip/active_set.zig

```diff
@@ -128,7 +128,7 @@ pub const ActiveSet = struct {
     }
 };
 
-test "gossip.active_set: init/deinit" {
+test "active_set init/deinit" {
```

Comment:
very minor but since we are in active_set would
```suggestion
test "init/denit" {
```
be more consitent